lib/bundler/audit/cli.rb

```diff
@@ -60,10 +61,10 @@ def check
       end
 
       desc 'update', 'Updates the ruby-advisory-db'
-      def update
+      def update(quiet = false)
```

Comment:
Could you rewrite this to use `options={}` style of passing in a `quiet: true` option?